test/regression/2020-04-27-stp-array-names.c

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang %s -emit-llvm %O0opt -g -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee -output-dir=%t.klee-out --search=bfs --solver-backend=stp --max-instructions=1000 %t.bc
```

Comment:
I would still add the z3 one xD, also don't you need `REQUIRE` for z3 only build?